### ISSUE TITLE
upgrade to fontawesome 5

### ISSUE THREAD

**Issue by jsichi** [jgrapht/jgrapht#962]
Our website uses fontawesome 4.  I wanted to use the new fa-python icon, which is only available in fontawesome 5, but the upgrade seemed like a bit of work, so I punted.

**Comment by morrme** [comment]
Are you referring to this process: https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4 ?

**Comment by jsichi** [comment]
Yes.  If you can submit a PR for this, that would be great.  Here's the info on how to test your changes locally (but you can skip the javadoc/hercule stuff since we only use fontawesome for the main site):

https://github.com/jgrapht/jgrapht/wiki/Dev-guide%3A-Website-Deployment#local-testing
